xml/art_sle_amazon_ecs.xml

```diff
@@ -41,21 +41,27 @@
     <para>
       Creating a new ECS cluster in Amazon is accomplished by using the
       <link xlink:href="https://console.aws.amazon.com/ecs/home#/firstRun">First
-      Run Wizard</link>
+      Run Wizard.</link> Alternatively you may use the
+      <emphasis role="strong">aws</emphasis> command line tools to
+      <link xlink:href=http://docs.aws.amazon.com/cli/latest/reference/ecs/create-cluster.html">create</link>
+      the cluster. In the First Run Wizard select
+      <emphasis role="strong">Custom</emphasis> if you want to create a
+      setup unique to your own needs. The
+      <emphasis role="strong">Amazon ECS sample</emphasis> selection creates
+      a pre-defined web application setup, that may be customized. By default
+      the First Run Wizard selects the Amazon ECS AMI, we will change this
+      later. Note that using the First Run Wizard also creates an autoscaling
+      group. This implies that at least one instance (default setting) is
+      always running and a new instance will be started by the autoscaling
+      group if the running instance should be stopped or terminated.
     </para>
     <para>
       We recommend retaining the default cluster name
-      <emphasis role="strong">default</emphasis>, because this allows for
-      automatically attaching an ECS instance to the cluster without
-      further configuration of the instance.
-    </para>
-    <para>
-      Renaming the cluster requires additional configuration once an ECS
-      instance is running. When the cluster name is
-      <emphasis role="strong">default</emphasis> the ECS instance will
-      automatically attach itself to the cluster. The additional configuration
-      steps for clusters with a name other than
-      <emphasis role="strong">default</emphasis> are described in the
+      <emphasis role="strong">default</emphasis>. When the cluster is named
+      <emphasis role="strong">default</emphasis> an ECS instance will
+      automatically attach itself to the cluster without additional
+      configuration steps. Selecting a custom name for the cluster
+      requires the instance to be configured as outlined in the
       <xref linkend="alternative-cluster-name"/> section below.
```

Comment:
In some cases, the `<xref/>` creates already the word "section". As such, I would rephrase the sentence to avoid that issue:

```
Selecting a custom name for the cluster requires the instance to be configured
as outlined in <xref linkend="alternative-cluster-name"/>.
```